beacon_node/lighthouse_network/gossipsub/Cargo.toml

```
@@ -0,0 +1,50 @@
+[package]
+name = "gossipsub"
+edition = "2021"
+description = "Sigma prime's version of Gossipsub protocol for libp2p"
+version = "0.5.0"
+authors = ["Age Manning <Age@AgeManning.com>"]
+license = "MIT"
+repository = "https://github.com/sigp/lighthouse/"
+keywords = ["peer-to-peer", "libp2p", "networking"]
+categories = ["network-programming", "asynchronous"]
+
+[features]
+wasm-bindgen = ["getrandom/js", "instant/wasm-bindgen"]
+
+[dependencies]
+async-channel = { workspace = true }
+asynchronous-codec = "0.7.0"
+base64 = "0.21.7"
+byteorder = "1.5.0"
+bytes = "1.5"
+either = "1.9"
+fnv = "1.0.7"
+futures = "0.3.30"
+futures-ticker = "0.0.3"
+futures-timer = "3.0.2"
+getrandom = "0.2.12"
+hex_fmt = "0.3.0"
+instant = "0.1.12"
+libp2p = { version = "0.53", default-features = false, features = ["identify", "yamux", "noise", "dns", "tcp", "tokio", "plaintext", "secp256k1", "macros", "ecdsa", "metrics", "quic", "upnp"]}
```

I think the `features` section can be removed.

```suggestion
libp2p = { version = "0.53", default-features = false }
```

---

thanks Akihito, updated!